bin/dump-versioned-schema.sh

```diff
@@ -4,6 +4,7 @@ DIRNAME=`dirname $0`
 
 if [ -z $2 ]; then
     echo "usage: $0 source_schema output_schema"
+    echo "exemple: louis_v005 to louis_v006"
```

Comment:
Example: $0 louis_v005 louis_v006

Reply:
Indeed, I'll fix that, thank for noticing 👍